content/en/docs/concepts/security/pod-security-policy.md

```diff
@@ -255,7 +258,7 @@ The name of a PodSecurityPolicy object must be a valid
 And create it with kubectl:
 
 ```shell
-kubectl-admin create -f example-psp.yaml
+kubectl-admin create -f https://k8s.io/examples/policy/example-psp.yaml
```

Comment:
You can skip “Define the example PodSecurityPolicy object in a file” too.

(nit: this tidying, to avoid having the reader paste in a manifest, would be even better if done in its own commit)

Reply:
https://github.com/kubernetes/website/pull/34675/commits/1d55061a5a748bd9cd953470c847ac012cc2cf8f